src/pages/candidate-details/candidate-details.analytics.effects.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs/observable/of';
+import { switchMap, map, withLatestFrom } from 'rxjs/operators';
+import { Store, select } from '@ngrx/store';
+import { StoreModel } from '../../common/store.model';
+import { getJournalState } from '../journal/journal.reducer';
+import { AnalyticsProvider } from '../../providers/analytics/analytics';
+import {
+    AnalyticsDimensionIndices,
+    AnalyticsScreenNames,
+    AnalyticsEventCategories,
+    AnalyticsEvents,
+  } from '../../providers/analytics/analytics.model';
```

Comment:
These imports aren't consistent with the ones below (closing brace is indented).

Reply:
removed indent on closing brace

Reply:
fixed